### ISSUE TITLE
CATS: Failure in backend_compatibility suite

### ISSUE THREAD

**Issue by fargozhu** [SUSE/kubecf#143]
**Describe the bug**

Seeing this when running CATS, with only the backend_compatibility test group enabled.

```
Summarizing 1 Failure:

[Fail] [backend_compatibility] Backend Compatibility An app staged on the DEA [BeforeEach] runs on Diego 
/var/vcap/packages/acceptance-tests/src/github.com/cloudfoundry/cf-acceptance-tests/backend_compatibility/backend_compatibility.go:54
```

```
runs on Diego

[2019-11-06 03:31:28.61 (UTC)]> cf oauth-token 
SILENCING COMMAND OUTPUT
[2019-11-06 03:31:29.62 (UTC)]> curl -k -H Expect: -s https://https://api.10.104.255.255.nip.io/v2/apps/f35245ce-b14a-4236-ac1e-4c41c5692fdb/droplet/upload -X PUT -F droplet=@assets/dora-droplet.tar.gz -H Authorization: [REDACTED] 
```

**To Reproduce**
N/A

**Expected behavior**
To pass

**Environment**
N/A

**Additional context**
N/A

**Comment by mook-as** [comment]
Should already be fixed by #126

**Comment by f0rmiga** [comment]
Confirmed it is fixed:
```
Ran 1 of 213 Specs in 68.170 seconds
SUCCESS! -- 1 Passed | 0 Failed | 0 Flaked | 1 Pending | 211 Skipped


Ginkgo ran 1 suite in 1m8.333111176s
Test Suite Passed
```
